packages/flutter_localizations/lib/src/l10n/generated_cupertino_localizations.dart

```diff
@@ -197,6 +197,182 @@ class CupertinoLocalizationFr extends GlobalCupertinoLocalizations {
   String get todayLabel => r'aujourd' "'" r'hui';
 }
 
+/// The translations for Portuguese (`pt`).
```

Comment:
the analyzer will give an error if this file wasn't autogenerated (instructions on top)